catalog/spatial/registry/registry-api/src/main/java/org/codice/ddf/registry/api/RegistryStore.java

```diff
@@ -32,4 +32,10 @@
      * @return true if allowed otherwise false
      */
     boolean isPullAllowed();
+
+    /**
+     * Indicates the Id associated with this registry
+     * @return registry id in a string
+     */
+    String getRegistryId();
```

Comment:
Is this API released? Do you need a default implementation to not break old implementations?